src/05-derivatives/05-diffusion-derivatives.md

```diff
@@ -0,0 +1,467 @@
+# Diffusion derivatives
+
+## Preprocessed diffusion weighted images
+
+Multiple different versions of preprocessing can be stored for the same source
+data. To distinguish them from each other, the `desc` filename keyword can be
+used. Details of preprocessing performed for each variation of the processing
+should be included in the pipeline documentation.
+
+```Text
+<pipeline_name>/
+    sub-<participant_label>/
+        dwi/
+            <source_keywords>[_space-<space>][_desc-<label>]_dwi.nii[.gz]
+            <source_keywords>[_space-<space>][_desc-<label>]_dwi.bvals
+            <source_keywords>[_space-<space>][_desc-<label>]_dwi.bvecs
+            <source_keywords>[_space-<space>][_desc-<label>]_dwi.json
+```
+
+The JSON sidecar file is REQUIRED (due to the REQUIRED `SkullStripped` field -
+see [Common Data Types](02-common-data-types.md)), and if present can be used to
+store information about what preprocessing options were used (for example
+whether denoising was performed, corrections applied for field inhomogeneity /
+gradient non-linearity / subject motion / eddy currents, intensity normalization was
+performed, etc.).
+
+Additional reserved JSON metadata fields:

```

Comment:
This second part belongs in the Common Derivatives section, IMHO. cc/ @effigies @sappelhoff